plugin-server/src/cdp/templates/_destinations/google_sheets/google_sheets.template.ts

```diff
@@ -0,0 +1,90 @@
+import { NativeTemplate } from '~/cdp/types'
+
+export const template: NativeTemplate = {
+    free: false,
+    status: 'beta',
+    type: 'destination',
+    id: 'native-google-sheets',
+    name: 'Google Sheets',
+    description: 'Update a Google Sheet with the incoming event data',
+    icon_url: '/static/services/google-sheets.svg',
+    category: ['Custom'],
+    perform: async (request, { payload }) => {
+        const { spreadsheet_id, spreadsheet_name, data_format, fields } = payload
+        const rowData = Object.values(fields)
+
+        const url = `https://sheets.googleapis.com/v4/spreadsheets/${spreadsheet_id}/values/${spreadsheet_name}:append?valueInputOption=${data_format}`
```

Comment:
tbh this is an example of where I think this would have been just as easy as a hog function... Its basically a simple fetch it seems... 

Any reason why this one is native?